components/ui/Tag.jsx

```diff
@@ -0,0 +1,15 @@
+export default function Tag({ fill, children }) {
+  return (
+    <>
+      {fill ? (
+        <p className="bg-grey-200 text-lg leading-18 tracking-tight text-grey-100 font-normal w-fit px-2.5 py-0.6 rounded-md">
```

Comment:
Tailwind has a default line height for ratio 1 called `leading-none`. you could change the leading-18 class to the leading-none class.